doc/zstd_compression_format.md

```diff
@@ -1083,7 +1083,8 @@ It depends on :
   Presuming an `Accuracy_Log` of 8,
   and presuming 100 probabilities points have already been distributed,
   the decoder may read any value from `0` to `256 - 100 + 1 == 157` (inclusive).
-  Therefore, it must read `log2sup(157) == 8` bits.
+  Therefore, it may read up to `log2sup(157) == 8` bits, where `log2sup(N)`
+  is the smallest integer `T` that satisfies `(1 << T) > N`.
```

Comment:
I think it's actually `(1 << T) >= N` (note the `>=`)

Reply:
Yes you are right

It depends on how is `N` defined.
If `N` is the alphabet size, then a size of `256` implies that the highest possible value is `255` since the alphabet starts at `0`. In which case, 8-bits are enough.
But if `N` is the highest possible value, then indeed, 9 bits are required to encode `256`.

In the example above, `N` takes the place of `157`, which is defined as the last value inclusive.
So it's a `<`.